### PR TITLE
Fixes and renames Image.vue. Edits to App.vue

### PR DESCRIPTION
**Pull request by The-Mills**
Vue does not like when the custom template component is named "Image". Hence, everything is now "ImageDisplayer"
App.vue now preloads all images using require(), instead of relying on other components to load them from URL strings.
Made the code in these two files better-structured and easier to read.

### CODE REVIEW

File: src/components/ImageDisplayer.vue [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<template>
+  <v-parallax :src="imgSrc" :height="displayHeight">
+
+	<v-row       v-if="headerTxt||txt"                align="center"   justify="center">
+        <v-col   class="text-center"                  cols="12">
+          <h2    v-if="headerTxt">{{headerTxt}}</h2>
+          <p     v-if="txt">{{txt}}</p>
```
Comment (maxwildersmith) [inline src/components/ImageDisplayer.vue:7]: What happened to the spacing?

File: src/components/ImageDisplayer.vue [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<template>
+  <v-parallax :src="imgSrc" :height="displayHeight">
```
Comment (maxwildersmith) [inline src/components/ImageDisplayer.vue:2]: Make it set the height attribute only if a displayHeight is set

File: src/components/ImageDisplayer.vue (hunk 1 above)
Comment (The-Mills) [reply src/components/ImageDisplayer.vue:7]: Tried spacing and vertically aligning parameters to make everything easier to read. Doesn't seem to turn out well so I'll just space it normally.

File: src/components/ImageDisplayer.vue (hunk 2 above)
Comment (The-Mills) [reply src/components/ImageDisplayer.vue:2]: Per the v-parallax API, the height defaults to 500 if no value is passed in. So you can omit displayHeight in your ImageDisplayer constructor as the code stands. I'll add a second one to App.vue as an example.

**Pull request merged**